backend/src/main/java/fk/prof/backend/http/BackendHttpVerticle.java

```diff
@@ -78,6 +92,11 @@ private Router setupRouting() {
     router.put(ApiPathConstants.BACKEND_PUT_ASSOCIATION)
         .handler(this::handlePutAssociation);
 
+    router.post(ApiPathConstants.BACKEND_POST_POLL)
```

Comment:
Fixed in https://github.com/Flipkart/fk-prof/commit/a5eb03869082b66fde0437b7c51549b359204631